tests/dateTime.py

```diff
@@ -54,74 +56,74 @@ def testLeapSecond(self):
+        self.assertEqual(ts.nsecs(), long(1192755506000000000))
         self.assertAlmostEqual(ts.get(DateTime.MJD, DateTime.UTC), 54392.040196759262)
         ts2 = ts
         self.assertEqual(ts, ts2)
-        ts2 = DateTime(1192755473000000000L, DateTime.UTC)
+        ts2 = DateTime(long(1192755473000000000), DateTime.UTC)
         self.assertEqual(ts, ts2)
-        ts2 = DateTime(1234567890000000000L, DateTime.UTC)
+        ts2 = DateTime(long(1234567890000000000), DateTime.UTC)
         self.assertNotEqual(ts, ts2)
 
     def testBoundaryMJD(self):
         ts = DateTime(47892.0, DateTime.MJD, DateTime.UTC)
-        self.assertEqual(ts.nsecs(DateTime.UTC), 631152000000000000L)
-        self.assertEqual(ts.nsecs(DateTime.TAI), 631152025000000000L)
+        self.assertEqual(ts.nsecs(DateTime.UTC), long(631152000000000000))
+        self.assertEqual(ts.nsecs(DateTime.TAI), long(631152025000000000))
         self.assertEqual(ts.get(DateTime.MJD, DateTime.UTC), 47892.0)
 
     def testCrossBoundaryNsecs(self):
-        ts = DateTime(631151998000000000L, DateTime.UTC)
-        self.assertEqual(ts.nsecs(DateTime.UTC), 631151998000000000L)
-        self.assertEqual(ts.nsecs(DateTime.TAI), 631152022000000000L)
+        ts = DateTime(long(631151998000000000), DateTime.UTC)
+        self.assertEqual(ts.nsecs(DateTime.UTC), long(631151998000000000))
+        self.assertEqual(ts.nsecs(DateTime.TAI), long(631152022000000000))
 
     def testNsecsTAI(self):
-        ts = DateTime(1192755506000000000L, DateTime.TAI)
-        self.assertEqual(ts.nsecs(DateTime.UTC), 1192755473000000000L)
-        self.assertEqual(ts.nsecs(DateTime.TAI), 1192755506000000000L)
-        self.assertEqual(ts.nsecs(), 1192755506000000000L)
+        ts = DateTime(long(1192755506000000000), DateTime.TAI)
+        self.assertEqual(ts.nsecs(DateTime.UTC), long(1192755473000000000))
+        self.assertEqual(ts.nsecs(DateTime.TAI), long(1192755506000000000))
+        self.assertEqual(ts.nsecs(), long(1192755506000000000))
         self.assertAlmostEqual(ts.get(DateTime.MJD, DateTime.UTC), 54392.040196759262)
 
     def testNsecsDefault(self):
-        ts = DateTime(1192755506000000000L)
-        self.assertEqual(ts.nsecs(DateTime.UTC), 1192755473000000000L)
-        self.assertEqual(ts.nsecs(DateTime.TAI), 1192755506000000000L)
-        self.assertEqual(ts.nsecs(), 1192755506000000000L)
+        ts = DateTime(long(1192755506000000000))
+        self.assertEqual(ts.nsecs(DateTime.UTC), long(1192755473000000000))
+        self.assertEqual(ts.nsecs(DateTime.TAI), long(1192755506000000000))
+        self.assertEqual(ts.nsecs(), long(1192755506000000000))
         self.assertAlmostEqual(ts.get(DateTime.MJD, DateTime.UTC), 54392.040196759262)
 
     def testNow(self):
         successes = 0
-        for i in xrange(10):       # pylint: disable-msg=W0612
+        for i in range(10):       # pylint: disable-msg=W0612
```

Comment:
No idea. Added by @ktlim back in commit 9a8ab201bc5b1828631fec02213a4460b6a9ee6b